aosp_diff/base_aaos/frameworks/native/0001-Add-the-support-for-Vulkan-1.3.patch

```diff
@@ -0,0 +1,41 @@
+From af790dd7953e4ddc77a761d76ab3f6770ab0fe7f Mon Sep 17 00:00:00 2001
+From: "Li, Gaoshun" <gaoshun.li@intel.com>
+Date: Tue, 6 Feb 2024 12:02:26 +0000
+Subject: [PATCH] Add the support for Vulkan 1.3
```

Comment:
As i understood from comment it is a work around or hack , please tag it with "[WA]" in patch subject and commit msg.

Reply:
this is not WA, Android S only support Vulkan 1.1, while our driver already support Vulkan 1.3